tests/python-gpu/test_from_columnar.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import xgboost as xgb
+import sys
+import pytest
+sys.path.append("tests/python")
+import testing as tm
+
+pytestmark = pytest.mark.skipif(**tm.no_cudf())
+
+
+class TestFromColumnar:
+    '''Tests for constructing DMatrix from data structure conforming Apache
+Arrow specification.'''
+
+    @pytest.mark.skipif(**tm.no_cudf())
+    def test_from_cudf():
```

Comment:
I think I could really use some help to setup test. ;-)

Reply:
Is there anything we need to change in the Jenkins CI?